src/components/AssignmentForm.js

```diff
@@ -0,0 +1,60 @@
+import React ,{useState, useEffect} from 'react'
+import TextField from '@material-ui/core/TextField';
+
+const AssignmentForm =(props)=> {
+    const initialvalues ={
+        name: "",
+        data: "",
+    } 
+    var [values, setValues] = useState({})
+    useEffect(()=>{
+        if(props.currentId ==='')
+        setValues({
+            ...initialvalues
+        })
+        else
+        setValues({
+            ...props.assignment[props.currentId]
+        })
+
```

Comment:
I would have expected a `})` on this line to close out the `useEffect` function call on line 10, or is all the code below still part of the lambda function started on line 10?